.github/workflows/label.yml

```diff
@@ -0,0 +1,10 @@
+name: Labeler
+on: [issues, pull_request]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: DataBiosphere/azul-github-labeler-action@releases/v4
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
and this one are different to their counterparts in Azul. Why?

Reply:
This was caused by updating lines individually from the default template provided, instead of replacing the whole file.
I will add a fixup commit to match the Azul values.
![Screen Shot 2021-02-26 at 1 03 53 PM](https://user-images.githubusercontent.com/46407464/109355015-79f9a080-7833-11eb-9344-85a462313e48.png)